cloudferry/discover.py

```diff
@@ -176,6 +177,7 @@ def discover_all(cfg, cloud):
         LOG.debug('Finished discovering %s using %s',
```

Comment:
But you can't find any "results" there. This file is not designed to be viewed by human being.